checks.lua

```diff
@@ -297,21 +293,30 @@ function checkers.int64(arg)
     return false
 end
 
+local has_box = rawget(_G, 'box') ~= nil
+if has_box and box.tuple ~= nil then
+    checkers.tuple = box.tuple.is
+end
+
 local has_decimal, decimal = pcall(require, 'decimal')
-if has_decimal and decimal.is_decimal then
+if has_decimal then
     checkers.decimal = decimal.is_decimal
 end
 
-if is_tarantool() == true then
-    checkers.tuple = box.tuple.is
-
-    -- https://github.com/tarantool/tarantool/blob/7682d34162be34648172d91008e9185301bce8f6/src/lua/uuid.lua#L29
-    local uuid_t = ffi.typeof('struct tt_uuid')
-    function checkers.uuid(arg)
-        return ffi.istype(uuid_t, arg)
+local function add_ffi_type_checker(checks_t, ffi_obj)
```

Comment:
Nit: I would call the second argument `ctype`.

Reply:
Renamed